src/App.js

```diff
@@ -1,26 +1,38 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import style from './App.css';
+import { Provider} from 'react-redux';
+import {createStore, applyMiddleware, compose } from 'redux';
+import { restaurantsReducer } from './reducers/Reducers.js';
+import bkgImage  from './restaurant_image.jpg';
+import RestaurantsList from './containers/RestaurantsList.js'
+import thunk from 'redux-thunk';
+
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const store = createStore(restaurantsReducer, {
+  isFetching: false,
+  items: [],
+  error: undefined,
+  lastUpdate: undefined,
+},
+  composeEnhancers(applyMiddleware(thunk))
+);
 
 function App() {
   return (
-    <div className="App">
+      <Provider store={store}>
```

Comment:
indentation is off.
make sure you have configured `eslint` with some basics configs. ( i would recommend to take the eslintrc file from spacestation-web repo)

Reply:
still don't know why the change wasn't in the previous commits....

Reply:
check if you have soft tabs (spaces) or hard tabs (\t) set in you editor.
it might be two spaces and a tab.